progenitor-impl/tests/test_specific.rs

```diff
@@ -167,3 +167,36 @@ fn test_default_params() {
         &output,
     );
 }
+
+/// Test generated dependencies are correct
+#[test]
```

Comment:
this file has tests of specify types; I'd suggest that these tests don't belong here.

Reply:
maybe put these in a new file `test_deps.rs` , or broader `test_cargo_toml.rs` ?

Reply:
Sure